bindings/python/src/operator.rs

```diff
@@ -29,10 +29,9 @@ use crate::*;
 fn build_operator(
     scheme: ocore::Scheme,
     map: HashMap<String, String>,
-    blocking: bool,
 ) -> PyResult<ocore::Operator> {
     let mut op = ocore::Operator::via_map(scheme, map).map_err(format_pyerr)?;
-    if blocking && !op.info().full_capability().blocking {
```

Comment:
This means blocking layer will be enabled even for AsyncOperator, will it incur any performance penalty?

Reply:
Adding BlockingLayer here is zero-cost:

- We are using the async runtime provided by pyo3.
- Only enabled while `blocking == false`
  - Services support blocking is not changed
  - Calling on blocking API on not supported services will spwan futures on async runtime  instead. (This won't happen for `AsyncOperator`)